alf/environments/suite_carla.py

```diff
@@ -674,9 +674,9 @@ def __init__(self, world: carla.World):
         self._world = world
         self._map = world.get_map()
         self._waypoints = self._map.generate_waypoints(2.0)
-        self._vehicles = []
-        self._vehicle_dict = {}
-        self._vehicle_locations = {}
+        self._actors = []
```

Comment:
comment that actors include both vehicles and pedestrians?

Reply:
commented